This patch changes meson.build so it would work with meson v0.44.1
instead of >=v0.47. has_link_argument is the only method used that is
missing on the previous version of meson.

diff --git a/meson.build b/meson.build
index b6df98b3..7f12d472 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,7 @@
 project('fwupd', 'c',
   version : '1.2.3',
   license : 'LGPL-2.1+',
-  meson_version : '>=0.47.0',
+  meson_version : '>=0.44.1',
   default_options : ['warning_level=2', 'c_std=c99'],
 )

@@ -115,9 +115,9 @@ test_link_args = [
   '-Wl,-z,now',
 ]
 foreach arg: test_link_args
-  if cc.has_link_argument(arg)
+#  if cc.has_link_argument(arg)
     global_link_args += arg
-  endif
+#  endif
 endforeach
 add_global_link_arguments(
   global_link_args,
